.travis.yml

```diff
@@ -29,45 +18,21 @@ matrix:
 
 jobs:
   include:
-    # Coverage tracking is slow with pypy, skip it.
-    - env: TOXENV=pypy-xdist PYTEST_NO_COVERAGE=1
-      python: 'pypy2.7-6.0'
-    - env: TOXENV=pypy3-xdist PYTEST_NO_COVERAGE=1
-      python: 'pypy3.5-6.0'
-
-    - env: TOXENV=py34-xdist
-      python: '3.4'
-    - env: TOXENV=py35-xdist
-      python: '3.5'
-    - env: TOXENV=py36-xdist
-      python: '3.6'
-    - env: TOXENV=py37
     - &test-macos
       language: generic
       os: osx
       osx_image: xcode9.4
       sudo: required
       install:
         - python -m pip install --pre tox
+      before_script:
+        - test $(python -c 'import sys; print("%d%d" % sys.version_info[0:2])') = 27
       env: TOXENV=py27-xdist
     - <<: *test-macos
       env: TOXENV=py37-xdist
       before_install:
-        - brew update
-        - brew upgrade python
-        - brew unlink python
-        - brew link python
```

Comment:
The "brew update/upgrade" is very slow.
This uses pyenv instead to activate/use the 3.7.1 Python that should be available.